scripts/west_commands/fetchers/http.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import requests
```

Comment:
Just FYI, the usual import order in the west_commands directory is:

1. standard library
2. third party from pypi
3. third party from within zephyr

So this would instead be:

```py
from abc import ...

import requests
from west ...

from fetches.fetcher ...
```

Again I won't insist since you're going to maintain this, but consistency would be better here IMO

Reply:
Thank you! I will definitely fix this and remember it for the future.